solidity/test/token/HypnativeMovable.t.sol

```diff
@@ -31,21 +31,27 @@ contract MockITokenBridgeEth is ITokenBridge {
 
 contract HypNativeMovableTest is Test {
```

Comment:
Can we add a test to prove undercollateralization never happens?

Reply:
good idea